### PR TITLE
(finished) Inferior Weapons Patch

### PR DESCRIPTION
**Pull request by VMSolidus**
People have been talking recently about how relatively powerful the medieval weapons are in this game, and some ideas have been thrown around to see how their issues can be fixed. One such idea was to make the medieval weapons far less likely to spawn, which would encourage players to make their own. However, I also had the helpful idea of making "inferior" versions of the Medieval Weapons, that are not outright fakes, but just so happen to be considerably less durable.

This patch seeks to help counterbalance the Medieval Weapons mod by introducing a small number of medieval weapons that are made out of low grade steel. They are indistinguishable; statwise, from the actual weapons, but their inferior material makes them less than battle-ready. Generally my idea is that in the modern day, the vast majority of weapons advertised as "Battle-ready" are often made from things like inferior grades of steel, and have very poor construction choices such as rat-tail tangs. 

I'll add in more weapons to use later, and likely update the welder to also repair Stainless Steel.

Done So Far(this list is more for me than anyone else really):
Wakizashi
katana
zweihander
jian
scimitar
longsword
arming sword 
battleaxe
cutlass
broadsword

### CODE REVIEW

File: data/json/item_groups.json [hunk 1]
```diff
@@ -5674,19 +5674,23 @@
...
-      ["cutlass", 10],
-      ["rapier", 3],
-      ["wakizashi", 5],
-      ["tanto", 7],
+      ["broadsword", 1],
+      ["zweihander", 1],
+      ["cutlass", 1],
+      ["rapier", 1],
+      ["wakizashi", 1],
+      ["jian_inferior", 30]
```
Comment (Malkeus) [inline data/json/item_groups.json]: Missed a comma here

File: data/json/item_groups.json [hunk 2]
```diff
@@ -5674,19 +5674,23 @@
...
+      ["scimitar_inferior", 25]
```
Comment (Malkeus) [inline data/json/item_groups.json]: And another here

File: data/json/item_groups.json [hunk 3]
```diff
@@ -5674,19 +5674,23 @@
...
+      ["longsword_inferior", 25]
```
Comment (Malkeus) [inline data/json/item_groups.json]: Here, 3.

File: data/json/item_groups.json [hunk 4]
```diff
@@ -5674,19 +5674,23 @@
...
+      ["arming_sword_inferior", 25]
```
Comment (Malkeus) [inline data/json/item_groups.json]: And 4.

**Comment by kevingranade** [comment]
I fully support your endeavor.  Real mideval weapons should be vanishingly
rare.

**Comment by Coolthulhu** [comment]
Are they supposed to be unrepairable?
This makes sense - if they were repairable, it would be tedious (not hard) to keep them working.

**Comment by evilexecutive** [comment]
I'm currently mulling over having them be repairable. I've also changed the material name to budget steel, as per a suggestion on the IRC. I'm not quite sure yet, as part of me feels like it would be an exercise in futility to try and get these weapons to retain their edge.

**Comment by Malkeus** [comment]
You can barely put an edge on bad steel, let alone maintain one. And it will be gone the first time you cut something with it.

**Comment by Coolthulhu** [comment]
Leaving them unrepairable would make the whole thing much more significant.
If they were repairable, you'd only need to solder them after more serious fights, which would defeat the whole purpose.

**Comment by Malkeus** [comment]
I like this. Suddenly hording all the things will be an asset and not just a compulsion. I'll break all the swords til I find one that isn't fake.

**Comment by Malkeus** [comment]
commas are the single most important thing in the entire game. next is brackets and braces.

**Comment by kevingranade** [comment]
1. The stats should be much lower than the real  weapons. (Don't have time
to check them right now, if they are, cool)
2. The difference between these and real weapons should be visible (again,
if they are, good).

**Comment by evilexecutive** [comment]
I'll go ahead and alter their stats to be just a few points shy of the real deal. They won't be -quite- as shitty as the fake ones made of foam and such. But I agree that they shouldn't be completely the same.

**Comment by Malkeus** [comment]
Try 
"//" : "This is a comment",
I'm not sure it will work in those types of entries

**Comment by SeanMirrsen** [comment]
Just a small point, wouldn't it make sense for museums to have a slightly higher chance of having a genuine item on display, rather than a fake or an imitation?

**Comment by Malkeus** [comment]
I agree with SeanMirrsen. If you're going to find the real deal anywhere, it should be in museum's And possibly the martial arts dojo's, HEMA enthusiasts have to practice somewhere, and chintzy swords wouldn't last long in a real demonstration fight.

**Comment by Malkeus** [comment]
If you're adding new materials, you should include some weapons that use them or change some existing weapons.

**Comment by evilexecutive** [comment]
For putting swords in Dojos, I probably wouldn't with the current one. If anything I would probably make a dedicated special area for HEMA Practice, and it would have a mixture of both practice boffers, and the real swords. 

Anyway, I'll do -some- tweaking of the museum chances. But Museums operate on a tight budget usually, and they typically don't mind comissioning cheaper fakes than the real deal. They would likely only have the real swords as rare donations.

**Comment by Malkeus** [comment]
Fair enough point about museum's. But there should be some place that they spawn more than others or it's just gonna be a crapshoot. I like targeting particular buildings to raid in the middle of a zombie infested town. Right now pawn shops and antique shops are high on the list.

**Comment by evilexecutive** [comment]
I believe mansions should be the most likely place to find genuine weapons, seeing as how the people who occupy them are more than willing to throw away money on a completely real sword that they can brag about. I'll go ahead and also add in the Macuahuitl

**Comment by SeanMirrsen** [comment]
Yeah, I doubt many Dojos would carry real weapons anyway. The whole point of training is not having to deal with the chance to inflict grievous wounds, so on the overall you have about the same chance of encountering a real weapon there.

The changes to museum drop rates look good enough from here.

**Comment by evilexecutive** [comment]
I just checked, the medieval item group(Which I believe to be used for mansions), is unchanged. I might add a 50% chance for items there to be the inferior quality, just for balance.

**Comment by Malkeus** [comment]
I'd forgotten how brutal the Macuahuitl looks. That covers a use for obsidian. Are you planning to use high carbon steel for the authentic medieval weapons?

**Comment by evilexecutive** [comment]
I'm actually planning on using the High Carbon steel for "advanced" weapons that are decently rare, as well as a few other purposes. The idea being that they cannot be replaced, as the player cannot manufacture them, only find them. And once they're gone(Which will take awhile), they're gone for good.

**Comment by Malkeus** [comment]
Nifty, I look forward to seeing that. YOu might want to consider doing another PR though, this ones creeping beyond the scope of the original intent.

**Comment by SeanMirrsen** [comment]
I wonder, is there a way to represent the "sudden failure" common to high-carbon-steel swords? Like, they can take abuse all you want, and so long as you periodically sharpen the cutting edge they'll stay perfect, but one particularly good hit against something hard will cause them to outright snap in half, because the material is all hardness and not enough elasticity.

I mean. You rarely ever see a bent katana. (I mean *bent*, not curved. :P) It's either near-pristine or snapped in twain.

**Comment by evilexecutive** [comment]
Yea, I'm pretty much done here with what I set out to accomplish with this mod. I'll do the last changes for the mansion drop rate on medieval weapons(50% chance of being inferior), and then call it a night for this.

**Comment by Malkeus** [comment]
Looks real good, thanks for this :)

**Comment by evilexecutive** [comment]
You're welcome! I'm glad to be doing some good work with my first time at github

**Comment by RadaRadaRada** [comment]
I see that the displayed names of inferior versions are identical to those of original ones. Diamond weapon recipes would confuse the player if he doesn't have the right version of weapon to be diamonised. Perhaps, original or inferior weapons could be renamed? Fake katana - katana - masterwork katana, or something like that?

And, perhaps, inferior versions could be used as alternatives to original one to create diamond weapons too. It's all about the sharpness and maintaining the edge, right?

**Comment by Malkeus** [comment]
As I understood it, the intent of this was to have identical weapons which were built with inferior materials and contained design flaws that made them inferior to the authentic version. We can distinguish them by looking at the material entry in the description, but they have the same name as the authentic, more durable version. I like the sneakiness of it, but others may not. 

And Kevin Granade said this a bit further up this thread: "2. The difference between these and real weapons should be visible (again,
if they are, good)." So it looks like you'll need to make them stand out some how. Maybe a subtle ® at the end of the name ehehe.

**Comment by Malkeus** [comment]
It [works](http://puu.sh/mknki/1605847d5a.png)! Look closely at the machete

**Comment by SeanMirrsen** [comment]
Aren't there.. like, already fake weapons that are identical in name to the genuine ones? The fakes have a line in the description identifying them as such, but their display name is the same, because otherwise there is no point.

For these inferior weapons, you could add something different, like a short description of an ID number/brand logo engraved on the weapon by the replica manufacturer. Or something.

I would, by the way, like to point out that having items that display a "category" until they have been picked up by the player (and maybe examined), would bring this game much closer to being a *roguelike*. As in, the display name would be one of a "category" assigned to the item, unless particular requirements have been met. I would welcome that direction very much, but I've been left with the impression that it's not a direction intended by the project management.

**Comment by Malkeus** [comment]
You mean 'sword' when you look at it from 10 tiles away, and then you pick it up and study it for a moment and it becomes 'katana'? No thank you.

**Comment by SeanMirrsen** [comment]
>You mean 'sword' when you look at it from 10 tiles away, and then you pick it up and study it for a moment and it becomes 'katana'? No, thank you.

It's arguably a bad idea to generalize that much, unless you're playing a character with low Int and no cutting weapon skills. But you would not be able to tell whether a "katana" is a fake, a modern replica, or a genuine item unless you picked it up and examined it, or had a lot of experience with them.

As a (potentially bad) example, would you know how to tell an AK-47 from an AK-74 without picking one up and examining it? You'd recognize a Kalashnikov, but there are so many varieties that it's often hard to tell them apart, and you'd never figure out what ammo it has loaded or what internal modifications it has regardless of how often you've seen one. The current "look" tool is way too omniscient.

**Comment by Coolthulhu** [comment]
Fake names are a bad mechanic. For them to actually work like in roguelikes with identification, we'd have to hide all the details like attack damage, move cost and materials.

**Comment by SeanMirrsen** [comment]
Funny thing is, we already do that.

For books.

**Comment by Coolthulhu** [comment]
But that isn't a reason to make it worse.
It doesn't work for books, it probably wouldn't work for weapons.
If anything, it would be better to just scrap the mechanic for book ID until we have randomized books or something like that.

**Comment by kevingranade** [comment]
The fake ones are supposed to be just what you're describing, decorative
weapons made in roughly the correct shape, but out of unsuitable materials.
Are these just more of the same?

**Comment by evilexecutive** [comment]
The general gist of these is that somewhere, a company tried to make items intended for novice sword collectors. But they realized that there were plenty of things they could get away with in the means of making money. While not decorative pieces like the aluminium 'fake' swords, these 'inferior' swords were somehow originally intended for combat, but got made completely wrong in the name of profit. 

This patch is mainly to dilute out the spawn groups for medieval weapons, to make it considerably harder for players to find actual swords. Ideally if a player gets sick of finding all the fake ones made before the cataclysm, they should suck it up and deal with an inferior sword, or try and study manuscripts to make one of their own. It stops the game from getting pretty broken early on, without outright just nerfing the medieval weapons.

**Comment by Malkeus** [comment]
I'm now picturing all the hopes that will be raised and then subsequently dashed by this...I love it ;)

**Comment by chaosvolt** [comment]
So we seriously made a massive rebalance by Coolthulhu to fit the medieval items into a specific gameplay niche, only to decide to break that.

**Comment by Skidborg** [comment]
I am really, really not a fan of things retaining the same display name when they are not the same item.  It messes with the utility of inventory management and area search menus, long after the player is well aware of the difference between the two items.

Perhaps set up the names with "fake", "replica" and "authentic" levels?

**Comment by chaosvolt** [comment]
Here's a thought. You know how shrooms have their actual effects obfuscated unless the player has enough survival skill? Why not do the same from replicas, with melee, fabrication, or some other relevant skill as 

However, one suggestion to enhance this. Determining an item to be fake should ideally update the display name in addition to the normal change in description.

**Comment by Coolthulhu** [comment]
> Why not do the same from replicas, with melee, fabrication, or some other relevant skill as 

Because it is a bad mechanic when there are multiple tricks to bypass it.
* Crafting menu will tell apart replica broadsword from regular one with blade recipe
* Inferior material will reveal it easily
* Trying to solder it will reveal it instantly

Then there are problems like the price, hiding info from the player who already knows whether it is a replica or the real deal and NPCs knowing what's up anyway.

> Perhaps set up the names with "fake", "replica" and "authentic" levels? 

+1 for the fake and replica tags. Player can easily tell them apart by stats, no need to require looking at stats.
Authentic could be just tag-less, though.

File: data/json/items/melee.json [hunk 5]
```diff
@@ -3349,7 +3349,57 @@
...
+    "name": "Macuahuitl",
+    "description": "This is an ancient sword that was once used by the aztecs. Its edge menaces with shiny flakes of obsidian. Even the gentlest of touches still breaks your skin.",
+    "price": 800000,
+    "material": "obsidian, wood",
+    "flags" : ["CHOP", "SHEATH_SWORD"],
+    "techniques": "WBLOCK_1",
+    "weight": 1133,
+    "volume": 7,
+    "bashing": 8,
+    "cutting": 45,
```
Comment (Coolthulhu) [inline data/json/items/melee.json]: Obsidian isn't really a monomolecular high-tech material, it is just volcanic glass that can be sharpened really well.

File: data/json/items/melee.json [hunk 6]
```diff
@@ -3349,7 +3349,57 @@
...
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": { "type": "knife" },
+    "category" : "weapons",
+    "qualities": [["CUT", 4], ["BUTCHER", 9]]
```
Comment (Coolthulhu) [inline data/json/items/melee.json]: Cut 4? While tool qualities don't really matter much, I don't see a club with blades on the sides being useful for cutting stuff up.

File: data/json/items/melee.json [hunk 7]
```diff
@@ -3349,7 +3349,57 @@
...
+  },{
+    "id": "macuahuitl",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "Macuahuitl",
+    "description": "This is an ancient sword that was once used by the aztecs. Its edge menaces with shiny flakes of obsidian. Even the gentlest of touches still breaks your skin.",
+    "price": 800000,
+    "material": "obsidian, wood",
+    "flags" : ["CHOP", "SHEATH_SWORD"],
```
Comment (Coolthulhu) [inline data/json/items/melee.json]: It won't fit in a sheath. Too broad and irregular.

File: data/json/materials.json [hunk 8]
```diff
@@ -522,6 +522,63 @@
...
+        "cut_resist" : 2,
+        "bash_dmg_verb" : "dented",
+        "cut_dmg_verb" : "scratched",
+        "dmg_adj" : [
+            "marked",
+            "dented",
+            "smashed",
+            "shattered"
+        ],
+        "acid_resist" : 15,
```
Comment (Coolthulhu) [inline data/json/materials.json]: Acid resist should stay sane - steel has 7.

File: data/json/materials.json [hunk 9]
```diff
@@ -522,6 +522,63 @@
...
+        "cut_resist" : 4,
+        "bash_dmg_verb" : "dented",
+        "cut_dmg_verb" : "scratched",
+        "dmg_adj" : [
+            "marked",
+            "dented",
+            "smashed",
+            "shattered"
+        ],
+        "acid_resist" : 25,
```
Comment (Coolthulhu) [inline data/json/materials.json]: 10 is enough.

File: data/json/materials.json [hunk 10]
```diff
@@ -675,6 +732,25 @@
...
+        "cut_resist" : 1,
+        "bash_dmg_verb" : "dented",
+        "cut_dmg_verb" : "scratched",
+        "dmg_adj" : [
+            "marked",
+            "dented",
+            "smashed",
+            "shattered"
+        ],
+        "acid_resist" : 50,
```
Comment (Coolthulhu) [inline data/json/materials.json]: No need for an overkill, 10 is fine.

File: data/json/materials.json [hunk 11]
```diff
@@ -522,6 +522,63 @@
...
+        "elec_resist" : 0,
+        "fire_resist" : 3,
+        "chip_resist" : 30, 
+        "density" : 30 
+    },{
+        "type" : "material",
+        "ident" : "budget_steel",
+        "name" : "budget steel",
+        "bash_resist" : 4,
+        "cut_resist" : 2,
```
Comment (Coolthulhu) [inline data/json/materials.json]: Is this budget steel supposed to be closer to wood than to iron? Iron is 4/4, even cheap steel should not go below that.
Why lowered cut resistance?

**Comment by SeanMirrsen** [comment]
>+1 for the fake and replica tags. Player can easily tell them apart by stats, no need to require looking at stats.
Authentic could be just tag-less, though.

"Tell apart", yes. "Figure out which one they found when they have nothing to compare to", not so much. I really, really don't like it when the game promotes acquiring out-of-character knowledge.

**Comment by Coolthulhu** [comment]
Too bad, because the descriptions and material field pretty clearly betray the inferiority already.
So we're left with an annoyance (misleading names) that serves no purpose, but pretends to have one.

File: data/json/items/melee.json [hunk 12]
```diff
@@ -3349,7 +3349,57 @@
...
+    "use_action": { "type": "knife" },
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 9]]
+  },{
+    "id": "macuahuitl",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "Macuahuitl",
+    "description": "This is an ancient sword that was once used by the aztecs. Its edge menaces with shiny flakes of obsidian. Even the gentlest of touches still breaks your skin.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: Double spaces after periods please.

File: data/json/items/melee.json [hunk 13]
```diff
@@ -3349,7 +3349,57 @@
...
+    "price": 800000,
+    "material": "obsidian, wood",
```
Comment (illi-kun) [inline data/json/items/melee.json]: Are you sure this way of material definition works?

File: data/json/items/melee.json [hunk 14]
```diff
@@ -3375,6 +3425,33 @@
...
         "category" : "weapons"
     },
+      {
+    "id": "cutlass_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "dark_gray",
+    "name": "cutlass",
+    "name_plural": "cutlasses",
+    "description": "This is a broad saber known for its use by sailors and pirates. This sword appears to be made very poorly, but it should still stand up to a few swings.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: double spaces.

File: data/json/items/melee.json [hunk 15]
```diff
@@ -3400,6 +3477,32 @@
         "category" : "weapons"
     },
     {
+    "id": "katana_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "katana",
+    "description": "This is a rare sword from Japan. While it's got the right edge and weight, the pommel just snaps off, and the blade seems pretty worn.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: Double spaces required.

File: data/json/items/melee.json [hunk 16]
```diff
@@ -3497,6 +3626,33 @@
     "category" : "weapons"
     },
     {
+    "id": "wakizashi_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "wakizashi",
+    "name_plural": "wakizashi",
+    "description": "This is a comparatively-common Japanese short sword. There's something not quite right about this sword.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: 2x spaces.

File: data/json/items/melee.json [hunk 17]
```diff
@@ -3545,6 +3701,31 @@
     "category": "weapons"
     },
     {
+    "id": "battleaxe_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "battle axe",
+    "description": "This is a huge axe designed for warfare. You notice that the axe heads aren't seated properly at all.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: 2x spaces.

File: data/json/items/melee.json [hunk 18]
```diff
@@ -4066,5 +4247,114 @@
...
-    }
+    },
+    {
+    "id": "jian_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "jian",
+    "name_plural": "jian",
+    "description": "This is an ancient Chinese doubled-edged straight sword. The sword is quite worn, and bent at an odd angle.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: 2x spaces.

File: data/json/items/melee.json [hunk 19]
```diff
@@ -4066,5 +4247,114 @@
...
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 8]]
+    },
+    {
+    "id": "scimitar_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a curved sword associated with various middle eastern and central asian countries. This one seems oddly dull and worn.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: 2x spaces.

File: data/json/items/melee.json [hunk 20]
```diff
@@ -4066,5 +4247,114 @@
...
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 8]]
+  },
+  {
+    "id": "arming_sword_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "arming sword",
+    "description": "This is a classic medieval sword, just the right size to use one-handed. This one doesn't seem to have been made right.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: 2x spaces.

**Comment by SeanMirrsen** [comment]
Don't know how correct it is to ask, but I have to wonder - *why double spaces*? I mean, beyond the "apparently that's the way it's done" thing, because I've no idea if some text formatting mechanic depends on it or it's just a thing that you people do.

**Comment by illi-kun** [comment]
@SeanMirrsen please see discussion in #5414. Summary is

> Single space after periods is for proportional fonts.
Double space after periods is for fixed-width fonts.
We use fixed-width fonts.

**Comment by chaosvolt** [comment]
That summary technically does not explain the actual WHY, but as far as I can tell the reasoning is for readability and (secondarily) to maintain consistency.

**Comment by kevingranade** [comment]
To make sentence breaks stand out. This has been the standard English
practice for typesetting literally as long as there has been a standard for
typesetting.

**Comment by chaosvolt** [comment]
Which would fall under readability, yeah. XP

**Comment by evilexecutive** [comment]
I've removed the content that didn't contribute to the original purpose of this PR, now it purely deals with Inferior Weapons type. I'd keep the other stuff to a seperate PR, if I ever decide to do anything with them.

**Comment by evilexecutive** [comment]
So, for putting up Replica/Fake for the Fake and Inferior Items, I honestly wouldn't bother with that. The point being that if you enter a building and just give a cursory glance over the contents, you aren't making a close inspection of all the items in it. To that cursory glance, Fakes, Replicas, and genuine swords all look the same to you. It's when you pick it up and examine the item do the differences show up.

**Comment by Coolthulhu** [comment]
'V' takes zero time. It will reveal whether the items are fake or real.
The only things same names will cause is annoyance of having to read stats and not just the name, making meta knowledge more important. Gimmicks like that are good maybe for one playthrough, if that.

**Comment by chaosvolt** [comment]
Hmm. So I'm not sure if it'll really do anything except add a little annoyance. However, unlike some additions, the annoyance has already been present, via the cheap-ass aluminum versions.

So in practice the net effect is making the real deal less common relative to the knockoffs.

**Comment by Skidborg** [comment]
"It's already a little annoying."  Does not justify making it more so.  Having the fake and real items share a name is already a bad design.  This just makes it more so.

**Comment by chaosvolt** [comment]
True. But like I said, this is by no means the worst idea to date. :V

File: data/json/items/melee.json [hunk 21]
```diff
@@ -4066,5 +4222,114 @@
...
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 8]]
+    },
+    {
+    "id": "scimitar_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a curved sword associated with various middle eastern and central asian countries.  This one seems oddly dull and worn.",
```
Comment (illi-kun) [inline data/json/items/melee.json]: Please capitalize the "A" in "asian".

File: data/json/items/melee.json [hunk 22]
```diff
@@ -3425,6 +3503,32 @@
         "category" : "weapons"
     },
     {
+    "id": "zweihander_inferior",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "zweihänder",
+    "description": "This is a huge two-handed sword from Germany.  You're not quite sure it's supposed to be that bendy.",
+    "rarity": 30,
```
Comment (illi-kun) [inline data/json/items/melee.json]: I've never seen this parameter before, what's supposed to do?

Comment (Coolthulhu) [inline data/json/items/melee.json]: Probably an old value for old drop system.
Should be removed as it doesn't do anything nowadays.

Comment (illi-kun) [inline data/json/items/melee.json]: I found the same parameter in ```archery.json```, ```melee.json```, and ```tools.json``` and I'm going to remove it in separate PR.

File: data/json/items/melee.json (hunk 21 above)
Comment (Rivet-the-Zombie) [inline data/json/items/melee.json]: The M and E in Middle Eastern too.

Comment (chaosvolt) [inline data/json/items/melee.json]: That would be my fault. Doh. @_@

**Pull request merged**